### DISCUSSION TITLE
Muddle both email and text

### DISCUSSION THREAD

**Discussion by uldisrudzitis** [mokhosh/muddle#2] [Ideas]
Make it possible to muddle both link and text, example:
`<a href="mailto:{{ $user->email }}">{{ $user->email }}</a>`

Expected result
`<a href="{{ muddled link }}}">{{ muddled text }}</a>`

**Comment by mokhosh** [comment]
That's a good idea. 
I will accept a pr with tests.

**Comment by mokhosh** [comment]
Done https://github.com/mokhosh/muddle/pull/14
